include/openenclave/bits/evidence.h

```diff
@@ -136,26 +136,21 @@ extern const char* OE_REQUIRED_CLAIMS[OE_REQUIRED_CLAIMS_COUNT];
 #define OE_CLAIM_VALIDITY_UNTIL "validity_until"
 
 /**
- * SGX PCESVN of platform.
+ * Universal entity identity.
  */
-#define OE_CLAIM_SGX_PCE_SVN "sgx_pce_svn"
+#define OE_CLAIM_UE_ID "ue_id"
```

Comment:
why the underscore?  https://datatracker.ietf.org/doc/draft-ietf-rats-eat/ just has "ueid" not "ue-id".